addons/mail/static/src/models/rtc_controller.js

```diff
@@ -87,6 +98,10 @@ registerModel({
             readonly: true,
             required: true,
         }),
+        displayTitle: attr({
```

Comment:
I feel like `displayTitle` is not a meaningful name.

Reply:
What about `videoButtonText`?

Reply:
`screenSharingTogglerText`? `video` could be mistaken for camera